### ISSUE TITLE
How to specify a coordinates for analysis？

### ISSUE THREAD

**Issue by guitanj** [lightvector/KataGo#480]
use these two gtp commands? 
•	lz-analyze [player (optional)] [interval (optional)] KEYVALUEPAIR KEYVALUEPAIR ...
•	kata-analyze [player (optional)] [interval (optional)] KEYVALUEPAIR KEYVALUEPAIR ...

**Comment by lightvector** [comment]
Yes, use these commands. They analyze a position, not a specific move, but they will report the results for different moves.

If you want to analyze a specific move and no other move, play the move on the board and then use these commands to analyze the position after that move.

**Comment by guitanj** [comment]
got it.
Thx!
